src/aat/resources/features/F-026/S-074.td.json

```diff
@@ -0,0 +1,27 @@
+{
+  "title": "must return appropriate negative response when request does not provide valid authentication credentials",
+  "_guid_": "S-074",
+  "_extends_": "F-026_Test_Data_Base",
+
+  "specs": [
+    "an active profile in CCD",
+    "does not provide valid authentication credentials",
+    "has an HTTP-403 return code"
+  ],
+  "request": {
+    "pathVariables": {
+      "uid": "InvalidUID"
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_403_Response",
+    "headers": {
+      "Content-Length": "[[ANY]]",
+      "Content-Encoding": "gzip",
+      "Content-Type": "application/json;charset=UTF-8"
```

Comment:
You can move these 3 lines to the Test data base files as these are common in all the responses.

Reply:
Updated: in Test Data Base file
  "expectedResponse": {
    "headers": {
      "Content-Length": "[[ANY]]",
      "Content-Encoding": "gzip",
      "Content-Type": "application/json;charset=UTF-8"
    }